guarddog/analyzer/metadata/npm/empty_information.py

```diff
@@ -0,0 +1,20 @@
+""" Empty Information Detector
+
+Detects if a package contains an empty description
+"""
+import os.path
+from typing import Optional
+
+from guarddog.analyzer.metadata.empty_information import EmptyInfoDetector
+
+MESSAGE = "This package has an empty description on PyPi"
```

Comment:
```suggestion
MESSAGE = "This package has an empty description on NPM"
```

Reply:
Should be using a template in the base class. Will fix